api/swagger/server/restapi/certHandler.go

```diff
@@ -0,0 +1,226 @@
+package restapi
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha1"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/asn1"
+	"encoding/pem"
+	"flag"
+	"io/ioutil"
+	"log"
+	"math/big"
+	"os"
+	"path/filepath"
+	"runtime"
+	"time"
+)
+
+var userAndHostname string
+
+const rootName = "rootCA.pem"
+const rootKeyName = "rootCA-key.pem"
+
+type mkcert struct {
```

Comment:
you don't need that. here you copy/past mkcert code without removing what you really need.
Do you need help to clean all that ?

Reply:
I need this , this is the name of my CA certificate and my key CA 

We can rename it massaCa or something like that but i need this to load the CA